validator/testcases/regex.py

```diff
@@ -558,6 +558,67 @@ def tests(self):
             compat_type='error')
 
 
+@register_generator
+class Gecko47RegexTests(CompatRegexTestHelper):
+    """Regex tests for Firefox 47 updates."""
+
+    VERSION = FX47_DEFINITION
+
+    def tests(self):
+        yield self.get_test(
+            r'\bnsIX509CertDB\b',
+            'Most methods in nsIX509CertDB had their unused arguments removed.',
+            (
+                'Most methods in nsIX509CertDB had their unused arguments '
+                'removed. See %s for more information.' % BUGZILLA_BUG % 1241646
+            ),
+            log_function=self.err.warning,
+            compat_type='error')
+
+        yield self.get_test(
+            r'\b(listTokens|listSlots)\b',
+            'listTokens() and listSlots() now return nsISimpleEnumerator instead of nsIEnumerator.',
+            (
+                'listTokens() and listSlots() now return nsISimpleEnumerator instead '
+                'of nsIEnumerator. See %s for more information.' % BUGZILLA_BUG % 1220237
```

Comment:
Looking in mxr [1] they all seem valid to me. But I haven't looked into each. Looks like there are only a handful of add-ons affected by this.

[1] https://mxr.mozilla.org/addons/search?string=listSlots%28%29&find=&findi=&filter=%5E%5B%5E%5C0%5D*%24&hitlimit=&tree=addons